config/webpack.base.config.js

```diff
@@ -22,7 +22,12 @@ const baseConfig = {
     new MomentLocalesPlugin(),
     // provide much cleaner feedback while building on the command line
     new WebpackBar(),
-    new webpack.DefinePlugin({ MC_VERSION: JSON.stringify(require(path.resolve(basedir, 'package.json')).version) }),
+    new webpack.DefinePlugin({
+      MC_VERSION: JSON.stringify(require(path.resolve(basedir, 'package.json')).version),
+      'process.env.SENTRY_DSN': JSON.stringify(process.env.SENTRY_DSN),
+      'process.env.SUPPORT_URL': JSON.stringify(process.env.SUPPORT_URL),
+      'process.env.INQUIRY_EMAIL': JSON.stringify(process.env.INQUIRY_EMAIL)
```

Comment:
PS @VinneyJ , I've updated ☝🏽 to include `SENTRY_ENVIRONMENT`

Reply:
Cool on it @kilemensi

Reply:
The suggested code as is causes errors that pont to node modules during build time 
``` at Parser.pp$3.parseMaybeUnary (/Users/vincentotieno/Documents/projects/Media/web-tools/node_modules/webpack/node_modules/acorn/dist/acorn.js:2066:19)
    at Parser.pp$3.parseExprOps (/Users/vincentotieno/Documents/projects/Media/web-tools/node_modules/webpack/node_modules/acorn/dist/acorn.js:2010:21)
    at Parser.pp$3.parseMaybeConditional (/Users/vincentotieno/Documents/projects/Media/web-tools/node_modules/webpack/node_modules/acorn/dist/acorn.js:1993:21)
    at Parser.pp$3.parseMaybeAssign (/Users/vincentotieno/Documents/projects/Media/web-tools/node_modules/webpack/node_modules/acorn/dist/acorn.js:1968:21) {
  pos: 6,
  loc: Position { line: 1, column: 6 },
  raisedAt: 7
}
SyntaxError: Unexpected token (1:6)
    at Parser.pp$4.raise (/Users/vincentotieno/Documents/projects/Media/web-tools/node_modules/webpack/node_modules/acorn/dist/acorn.js:2825:15)
```
  
 But this version works perfectly
 

``` 
           'process.env': {
              'SENTRY_DSN': JSON.stringify(process.env.SENTRY_DSN),
              'SENTRY_ENVIRONMENT': JSON.stringify(process.env.NODE_ENV || 'development'),
              'SUPPORT_URL': JSON.stringify(process.env.SUPPORT_URL),
              'INQUIRY_EMAIL': JSON.stringify(process.env.INQUIRY_EMAIL),
      }
```

I dug deeper and realized omiting the `JSON.stringify()` causes the error @kilemensi

Reply:
Cool @VinneyJ ... my issue with JSON.stringify is if a value is `null`, it will return `'null'` which we don't want. So if you have to, may be use ` || ''`? i.e.
```js
           'process.env': {
              'SENTRY_DSN': JSON.stringify(process.env.SENTRY_DSN || ''),
              'SENTRY_ENVIRONMENT': JSON.stringify(process.env.NODE_ENV || 'development'),
              'SUPPORT_URL': JSON.stringify(process.env.SUPPORT_URL || ''),
              'INQUIRY_EMAIL': JSON.stringify(process.env.INQUIRY_EMAIL || ''),
           }
```

So that we end up with `''` instead of `'null'` when env vars are not set.

Reply:
Ooh I see...learning new stuff every day. Thanks for that.